ansible/ansible_collections/agentic/agent_configs/plugins/modules/deepseek_tui_mcp.py

```
@@ -0,0 +1,354 @@
+            raise ValueError(msg)
+    else:
+        url = params.get("url")
+        if not url:
+            msg = (
+                "url is required when transport=http "
+                f"name={params.get('name')!r} scope={params.get('scope')!r}"
+            )
+            raise ValueError(msg)
+
+
+def state_transition(changed: bool, existed_before: bool, state: str) -> str:
```

Ruff (0.15.12)

[warning] Boolean-typed positional argument in function definition

(FBT001)